opds_import.py

```diff
@@ -354,12 +356,13 @@ def extract_feed_data(self, feed, feed_url=None):
         # DataSource if it doesn't already exist. This way you don't have
         # to predefine a DataSource for every source of OPDS feeds.
         data_source = DataSource.lookup(
-            self._db, name=self.data_source_name, autocreate=True
+            self._db, name=self.data_source_name, autocreate=True,
+            offers_licenses=True
```

Comment:
This code can be used with data sources that don't offer licenses. If it's only the MetadataWranglerCoverageProvider, that's ok since its data source has already been created. But I think it could be confusing.